charts/tractusx-connector/templates/deployment-controlplane.yaml

```diff
@@ -68,7 +68,7 @@ spec:
           {{- else if .Values.vault.hashicorp.enabled }}
           image: "tractusx/edc-controlplane-memory-hashicorp-vault:{{ .Values.controlplane.image.tag | default .Chart.AppVersion }}"
           {{- else if .Values.vault.azure.enabled }}
-          image: "tractusx/edc-controlplane-memory:{{ .Values.controlplane.image.tag | default .Chart.AppVersion }}"
+          image: "tractusx/edc-runtime-memory:{{ .Values.controlplane.image.tag | default .Chart.AppVersion }}"
```

Comment:
I think this is a mistake or?

Reply:
nope. our images are now on docker, and once this PR hits `main`, the image that will be built will be available as `tractusx/edc-runtime-memory`

Reply:
Yes, but it's the wrong file ;-) You made the changes in the `tractusx-connector` helm chart and not in the `tractusx-connector-memory`

Reply:
ahhh wait, got it, sorry! My fault! you are right ;-)